### PR TITLE
Modified MAUI Chart getting started content

### PR DESCRIPTION
**Pull request by VimalaThirumalaiKumarSF3739**
**Description**
I have followed common getting started file format and updated the content as discussed with PLO. Modified the following the getting started file for the following controls

- Cartesian
- Circular
- Funnel
- Pyramid
- Polar
- Sunburst

Modified sunburst getting started image to resolve image size CI failure issues

### CODE REVIEW

File: MAUI/Cartesian-Charts/Getting-Started.md [hunk 1]
```diff
@@ -16,14 +16,29 @@ To get start quickly with our .NET MAUI Cartesian Chart, you can check the below
...
+## Prerequisites
 
-1. Create a new .NET MAUI application in Visual Studio.
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfCartesianChart to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Charts and then install it.
-3. To initialize the control, import the Chart namespace.
-4. Initialize [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+Before starting, ensure the following are set up:
 
-{% tabs %} 
+1. .NET 7 SDK or later is installed.
```
Comment (PaulAndersonS) [inline MAUI/Cartesian-Charts/Getting-Started.md:23]: [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)

File: MAUI/Cartesian-Charts/Getting-Started.md [hunk 2]
```diff
@@ -16,14 +16,29 @@ To get start quickly with our .NET MAUI Cartesian Chart, you can check the below
...
+2. A .NET MAUI development environment is ready with either Visual Studio 2022 (v17.3 or later) or VS Code. If using VS Code, make sure the .NET MAUI workload is installed and configured as per the instructions provided.
```
[the remark this answers is not in the document]
Comment (VimalaThirumalaiKumarSF3739) [reply MAUI/Cartesian-Charts/Getting-Started.md]: Updated the content

File: MAUI/Cartesian-Charts/Getting-Started.md (hunk 1 above)
Comment (VimalaThirumalaiKumarSF3739) [reply MAUI/Cartesian-Charts/Getting-Started.md]: Updated the link

File: MAUI/Cartesian-Charts/Getting-Started.md [hunk 3]
```diff
@@ -16,12 +16,29 @@ To get start quickly with our .NET MAUI Cartesian Chart, you can check the below
...
+## Prerequisites
 
-1. Create a new .NET MAUI application in Visual Studio.
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfCartesianChart to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Charts and then install it.
-3. To initialize the control, import the Chart namespace.
-4. Initialize [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+Before starting, ensure the following are set up:
+
+1. .NET 7 SDK (https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. A .NET MAUI development environment is ready with either Visual Studio 2022 (v17.3 or later) or VS Code. If using VS Code, make sure the .NET MAUI workload is installed and configured as per the instructions provided here.
```
Comment (PaulAndersonS) [inline MAUI/Cartesian-Charts/Getting-Started.md:24]: instructions provided [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)

Comment (VimalaThirumalaiKumarSF3739) [reply MAUI/Cartesian-Charts/Getting-Started.md]: Updated hyperlink

**Review by SyncfusionKarthikeyan — APPROVED** [review]
This is fine

**Review by PaulAndersonS — APPROVED** [review]

File: MAUI/Cartesian-Charts/Getting-Started.md [hunk 4]
```diff
@@ -16,12 +16,29 @@ To get start quickly with our .NET MAUI Cartesian Chart, you can check the below
...
-1. Create a new .NET MAUI application in Visual Studio.
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfCartesianChart to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Charts and then install it.
-3. To initialize the control, import the Chart namespace.
-4. Initialize [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+Before starting, ensure the following are set up:
+
+1. .NET 7 SDK (https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. A .NET MAUI development environment is ready with either Visual Studio 2022 (v17.3 or later) or VS Code. If using VS Code, make sure the .NET MAUI workload is installed and configured as per the instructions provided [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
+
+## Step 1: Create a New MAUI Project
```
Comment (PaulAndersonS) [inline MAUI/Cartesian-Charts/Getting-Started.md:26]: Step 1: Create a New MAUI Project

Visual Studio

Go to File > New > Project and choose the .NET MAUI App template.
Name the project and choose a location, then click Next.
Select the .NET framework version and click Create.
Visual Studio Code

Open the command palette by pressing `Ctrl+Shift+P`` and type .NET:New Project and enter.
Choose the .NET MAUI App template.
Select the project location, type the project name and press enter.
Then choose Create project

Comment (VimalaThirumalaiKumarSF3739) [reply MAUI/Cartesian-Charts/Getting-Started.md:26]: Updated the suggested content

**Pull request merged**